src/VSCodeExtension/src/tests/unit/control-structure.test.ts

```diff
@@ -0,0 +1,107 @@
+import "mocha";
+import * as assert from "assert";
+import { formatter } from "../../formatter/formatter";
+import { spaceAfterIf } from "../../formatter/rules/control-structure";
+
+describe("control structure formatter tests", () => {
+  it("with no rules, the code is unchanged", () => {
+    const code = "if(1 == 1){H(qs[0]);}";
+    const expectedCode = "if(1 == 1){H(qs[0]);}";
+
+    assert.equal(formatter(code, []), expectedCode);
+  });
+
+  describe("space after if rule", () => {
+    it("adds space after if statement", () => {
+      const code = "if(1 == 1){H(qs[0]);}";
+      const expectedCode = "if (1 == 1){H(qs[0]);}";
```

Comment:
Some of these test cases will need to be updated once other rules are implemented (such as requiring a space before and after open curly brace, ie `if (1 == 1) { H(qs[0]); }`. But that's ok to leave this as-is for now and expect those updates to be done by later rule authors.

Reply:
cc: @amrondonp

Reply:
The intention of this rule is only to handle the space after the if statement, We obviously need integration tests that test several rules at one but this specific rule only is concerned about the spacing after the if